mpu6050/mpu6050.c

```diff
@@ -290,7 +341,7 @@ static void mpu6050_exit(void)
 module_init(mpu6050_init);
 module_exit(mpu6050_exit);
 
-MODULE_AUTHOR("Andriy.Khulap <andriy.khulap@globallogic.com>");
```

Comment:
You can add  another MODULE_AUTHOR without removing the previous line.

Reply:
Only added by "MODULE_AUTHOR".